web/src/services/theme/reearthTheme/common/colors.ts

```diff
@@ -4,6 +4,10 @@ const colors = {
     white: "#FFFFFF",
     transparentBlack: "rgba(0,0,0,0.7)",
     transparentLight: "rgba(0,0,0,0.4)",
+    bg: {
+      1: "#1e2086",
+      2: "#df3013",
+    },
```

Comment:
I don't think these are used anywhere in this PR and also background colors are 99.999% unlikely to be common across themes.
Can you remove this and leave any theme changes for your next task please 🙏🏻